lib/ruby/truffle/cext/ruby.h

```diff
@@ -26,15 +26,25 @@ extern "C" {
 typedef void *ID;
 typedef void *VALUE;
 
-extern VALUE Qfalse;
-extern VALUE Qtrue;
-extern VALUE Qnil;
+VALUE get_Qfalse();
+VALUE get_Qtrue();
+VALUE get_Qnil();
```

Comment:
I think an extern variable escapes. I'm not even sure my `static void *ruby_cext;` below is going to PE.